deployments/warp_routes/oXAUT/production-deploy.yaml

```diff
@@ -4,6 +4,12 @@ avalanche:
     collateralToken: "0x5bE94B17112B8F18eA9Ac8e559377B467556a3c3"
   token: "0x30974f73A4ac9E606Ed80da928e454977ac486D2"
   type: xERC20
+base:
+  owner: "0x3949eD0CD036D9FF662d97BD7aC1686051c4aeBF"
+  ownerOverrides:
+    collateralToken: "0xa7eccdb9be08178f896c26b7bbd8c3d4e844d9ba"
```

Comment:
This will get removed once its live yeah?

Reply:
it will be updated. we need that field in order to check for collateralToken owner